_posts/2020-11-09-Basics-With-Bioawk.md

```diff
@@ -47,7 +47,7 @@ We will continue to use FASTA/FASTQ files as an example moving forward. As menti
 
 We can use Bioawk's built-in Awk functions to further split the comments into tabular data and print them with the following recipe:
 
-`$ bioawk -c fasta '{split($comment,a,"|"); print a}' example.fasta | head -n 10`
+`$ bioawk -c fasta '{split($comment,a,"&#124;"); print a}' example.fasta | head -n 10`
 
 The above splits the comment of the sequences into tabular chunks, using "|" as the delimiter to separate chunks, and places them into an array, "a" and then print the first 10 rows of data using `head`.
```

Comment:
This is the line causing the problem. The strikethroughs work now though.

Reply:
line 52

Reply:
https://2020f-usfca-btec-640.github.io/bioawk/2020/11/09/Basics-With-Bioawk.html

Reply:
Believe it should be fixed here now: 05e7559b3e7923cbb66f279804464aeedec7f38e

Reply:
@sdverma89 you need to submit a new PR with the fix so that I can merge it to fix the table pipe issue

Reply:
line 52 (the markdown text is still causing a problem, not the code)